README.md

```diff
@@ -0,0 +1,222 @@
+<a name="readme-top"></a>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
```

Comment:
- [ ] `required`
- [ ] Please add a `Video presentation`
- [ ] recorded a video presenting the features of the project.
- [ ] The maximum length of the video is 5 minutes.
- [ ] The video should contain a clear and objective description about the project in professional English. The topic and purpose of the project are made clear.
- [ ] The video should contain a demo of the project, highlighting all the project features.
- [ ] The video should contain a highlight about a piece of the code the student finds interesting or is proud of making.
- [ ] It's recommended that students use [Loom](https://www.loom.com/) to record to video but they can use any tool they want as long as they share an online link of the uploaded video.